modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -229,8 +230,14 @@ export default class Tile3DLayer extends CompositeLayer {
 
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
-    const {attributes, indices, modelMatrix, cartographicOrigin, material} = content;
-    const {_getMeshColor} = this.props;
+    const {attributes, indices, modelMatrix, cartographicOrigin, material, objectIds} = content;
+    const {
+      pickable,
+      pickFeatures,
+      autoHighlight,
+      highlightColor,
```

Comment:
`this.getSubLayerProps` already forwards the base layer picking props: https://deck.gl/docs/api-reference/core/composite-layer#getsublayerprops

Reply:
Removed them. Thanks.